nightfall-optimist/src/app.mjs

```diff
@@ -17,18 +16,21 @@ import {
   getContractAbi,
   debug,
 } from './routes/index.mjs';
+import { getAddressNonce } from './services/transaction-sign-send.mjs';
 
-const spec = swaggerJsDoc(options);
 const environment = config.ENVIRONMENTS[process.env.ENVIRONMENT] || config.ENVIRONMENTS.localhost;
 
 const app = express();
 
 const ethPrivateKey = environment.PROPOSER_KEY;
 const { address } = web3.eth.accounts.privateKeyToAccount(ethPrivateKey);
+const nonce = await getAddressNonce(address);
 
 app.set('ethPrivateKey', ethPrivateKey);
 app.set('ethAddress', address);
+app.set('nonce', nonce);
```

Comment:
Sorry, I didn't get the idea around this constant `nonce`.

Reply:
I think that it is needed to create a signed transaction for making a block if the current proposer is the actual running proposer.

Reply:
I guess @velkoskis that is what I was trying to do, but since I am not refreshing the value stored in the state of the app, this doesn't make much sense. Thank @israelboudoux , I will re-think this one (alongside nonce error-handling above).
Thanks again both

Reply:
We finally agreed to skip all nonce tracking for now and monitor behaviour